### PR TITLE
[proxima-core] #100 allow specifying access as list

### PR DESCRIPTION
**Pull request by je-ik**

### CODE REVIEW

**Comment by je-ik** [comment]
R: @dmvk @LesTR

File: core/src/test/resources/test-reference.conf [hunk 1]
```diff
@@ -100,67 +100,67 @@
       # this is testing only storage
       storage: "inmem:///proxima_events"
       type: primary
-      access: "commit-log"
+      access: commit-log
```
Comment (LesTR) [inline core/src/test/resources/test-reference.conf]: MINOR: Is this " removal needed? Can it be more backward compatible?

Comment (je-ik) [reply core/src/test/resources/test-reference.conf]: It is backwards compatible. That is feature of hocon, these two statements are actually equal, so this is just a cleanup.

**Comment by je-ik** [comment]
Closes O2-Czech-Republic#100

**Pull request merged**